src/core/utils/parse_cmdline.cpp

```diff
@@ -33,12 +33,73 @@
 
 #include "parse_cmdline.hpp"
 
-#include <limits>
 #include <string.h>
 
 #include "common/code_utils.hpp"
 #include "net/ip6_address.hpp"
 
+namespace {
+template <typename Type> class NumericLimits
+{
+};
+
+template <> class NumericLimits<int8_t>
+{
+public:
+    static constexpr int8_t Min() { return INT8_MIN; }
```

Comment:
Adding `void` ...Same for all other definitions...
```suggestion
    static constexpr int8_t Min(void) { return INT8_MIN; }
```

Reply:
Done.